build/dockerfiles/rhel.install.sh

```diff
@@ -8,7 +8,36 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all && \
+architecture=""
+case $(uname -m) in
+       x86_64) architecture="amd64" ;;
+       s390x) architecture="s390x";;
+esac
+if [ $architecture == "amd64" ]; then \
+       microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip skopeo && microdnf -y clean all; \
+elif [ $architecture == "s390x" ]; then \
+       microdnf install -y git findutils bash wget yum gzip tar python3-six python3-pip && microdnf -y clean all && \
+       microdnf install -y gcc gcc-c++ make automake; \
+       wget https://github.com/stedolan/jq/releases/download/jq-1.5/jq-1.5.tar.gz; \
+       tar xvzf jq-1.5.tar.gz; \
+       cd jq-1.5 || exit; \
+       ./configure && make && make install; \
+       cd "$HOME" || exit; \
+       export GOVERSION=1.13; \
+       wget https://dl.google.com/go/go$GOVERSION.linux-s390x.tar.gz; \
+       tar -xzf go$GOVERSION.linux-s390x.tar.gz; \
+       export GOROOT=$HOME/go; \
+       cp "$HOME"/go/bin/go /usr/bin/; \
+       cp "$HOME"/go/bin/gofmt /usr/bin/; \
+       export CC=gcc; \
+       microdnf install -y gpgme-devel libassuan-devel device-mapper-devel; \
+       export GOPATH=$HOME; \
+       mkdir -p "$GOPATH"/src/github.com/containers; \
+       cd "$GOPATH"/src/github.com/containers || exit; \
+       git clone https://github.com/containers/skopeo; \
```

Comment:
rather than build from source, could we use Fedora binary?

https://rpmfind.net/linux/rpm2html/search.php?query=skopeo&submit=Search+...&system=&arch=s390x

Reply:
Thanks for the suggestions. I will work on incorporating these changes.

Reply:
note that to use this in downstream Brew builds, we would need to prefetch the rpms and stash them in the dist-git repo's lookaside cache, if they don't exist in RHEL content set repos. 

A bit awkward but not impossible.